templates/nginx.conf.j2

```diff
@@ -19,6 +19,11 @@ server {
     {% elif env == 'production' %}
     listen {{ nginx_port }};
     server_name {{ server_name }};
+
+    {% if laravel_https_redirect -%}
+    return 301 https://{{ server_name }}$request_uri;
+    {%- endif %}
```

Comment:
I changed the https:// to $scheme:// as per https://www.digitalocean.com/community/questions/best-way-to-configure-nginx-ssl-force-http-to-redirect-to-https-force-www-to-non-www-on-serverpilot-free-plan-by-using-nginx-configuration-file-only